doc/admin-guide/files/sni.yaml.en.rst

```diff
@@ -176,6 +176,8 @@ tunnel_route              Inbound   Destination as an FQDN and port, separated b
                                     Protocol <proxy-protocol>` for more information on Proxy Protocol and how it is
                                     configured for |TS|.
 
+                                    Note that the match group number can be used in combination with the ``{inbound_local_port}`` and ``{proxy_protocol_port}`` literal strings.
+
```

Comment:
The code and this comment don't seem to agree.  The code is such that only one of `{inbound_local_port}` or `{proxy_protocol_port}` will be used with `{inbound_local_port}` taking precedence.  If both are specified, the `{proxy_protocol_port}` function is not run at all.  I recommend address this in the documentation and better handle the case where both strings are present (warning or error).

Reply:
yes, if both are present I also suggest adding an error as that seems not allowed. It should be either one `proxy_protocol_port` or `inbound_local_port` with a combination of the groups.

Reply:
Thanks for reviewing this and  good point on handling the case where both port variables are specified. I updated the doc to make it more explicit that only one can be specified. I also added a check in the code that would emit an error message when both are present.